lib/api/graphql.dart

```diff
@@ -9,7 +9,7 @@ class GraphQLUtility {
   GraphQLClient client;
 
   Link _createLink(String idToken) {
-    final authLink = AuthLink(getToken: () => "Bearer $idToken");
+    final authLink = AuthLink(getToken: () async => "Bearer $idToken");
```

Comment:
just wondering since idk but is there a diff by using async here?

Reply:
I think I just changed it when I was going through the flutter documentation. I can check to see if there is any need to change it if not then I can remove it again.